Source/Core/Core/IOS/IOSC.cpp

```diff
@@ -370,6 +370,12 @@ ReturnCode IOSC::Decrypt(Handle key_handle, u8* iv, const u8* input, size_t size
   return DecryptEncrypt(Common::AES::Mode::Decrypt, key_handle, iv, input, size, output, pid);
 }
 
+ReturnCode IOSC::CryptOFB(u8* key, u8* iv, u8* input, size_t size, u8* output)
+{
+  Common::AES::CryptOFB(key, iv, iv, input, output, size);
+  return IPC_SUCCESS;
+}
+
```

Comment:
One more... is there a reason you're routing this through IOSC? It's called in exactly once place, and the return value is ignored (and always IPC_SUCCESS). Seems to me like KDDownload() could just call Common::AES::CryptOFB() directly.